requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@1.1.6#egg=edx-submissions==1.1.6
+git+https://github.com/edx/edx-submissions.git@efischer/swallow_reset_signal#egg=edx-submissions==1.1.6
```

Comment:
Yeah, it turns out Travis builds go a lot more smoothly when you actually update require dependencies lol